Gruntfile.js

```diff
@@ -237,6 +237,18 @@ module.exports = function(grunt) {
     addIosrtcHookCmd: 'cp plugins/cordova-plugin-iosrtc/extra/hooks/iosrtc-swift-support.js hooks/iosrtc-swift-support.js',
 
     exec: {
+      buildChromeWebStoreUploads: {
+        // The first, uproxy-chrome.zip, is for team fishfood.
+        // The second two, which have had their keys stripped, are
+        // for uploading to the Chrome Web Store.
+        command: 'pushd build/dist && ' +
```

Comment:
Can you put this logic in a script? Then you can just call the script here. I think the less we have in this large Gruntfile the better and it's good to have the option to run it manually if we need to.